src/main/scala/exercises/06_Collections.scala

```diff
@@ -103,7 +106,10 @@ object Collections {
    * to implement in your own free-style way.
    */
   def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
+    l match {
+      case List() => false
```

Comment:
I really like how you did this. This is similar to what I tried, based on my limited knowledge of SML.